src/Command/DrupalUpdate/DrupalPackageUpdate.php

```diff
@@ -0,0 +1,234 @@
+    $this->io->note('Start checking of available updates..');
+    $this->infoPackageFilesPath = $this->packageInfo->getInfoFilesList();
+    $this->io->note('Preparing all packages detail list(package name, package type,current version etc.).');
+    try {
+      $this->packageInfo->getPackageDetailInfo($this->infoPackageFilesPath);
+    }
+    catch (AcquiaCliException $e) {
+      // @todo handle exception.
+    }
+    return $this->listOfPackageAvailableUpdates();
+  }
+
+  /**
+   * @return array
+   */
+  public function listOfPackageAvailableUpdates() {
+    $version_detail = $this->packageInfo->getPackageData();
+    $package_info_files = $this->infoPackageFilesPath;
+    $drupal_docroot_path = getcwd() . '/docroot';
+    $git_commit_message_detail = [];
+    $git_commit_message_detail[] = [
+          'Package Name',
+          'Package Type',
+          'Current Version',
+          'Latest Version',
+          'Update Type',
+          'Download Link',
+          'File Path'
+      ];
+    foreach ($version_detail as $package => $versions) {
+      if (!isset($versions['available_versions'])) {
+        continue;
+      }
+      $git_commit_message['package'] = $package;
+      $git_commit_message['package_type'] = $versions['package_type'];
+      $git_commit_message['current_version'] = isset($versions['current_version'])?$versions['current_version']:'';
+      $git_commit_message['latest_version'] = isset($versions['available_versions'])?$versions['available_versions']['version']:'';
+      $git_commit_message['update_notes'] = isset($versions['available_versions']['terms'])?$this->getUpdateType($versions['available_versions']['terms']['term']):'';
+      $git_commit_message['download_link'] = isset($versions['available_versions'])?$versions['available_versions']['download_link']:'';
+      if (isset($package_info_files[$package . '.info']) && strpos($package_info_files[$package . '.info'], ",") !== FALSE ) {
+        $package_info_files[$package . '.info']=explode(',', $package_info_files[$package . '.info']);
+      }
+      if (isset($package_info_files[$package . '.info']) && is_array($package_info_files[$package . '.info'])) {
+        $file_paths=[];
+        foreach ($package_info_files[$package . '.info'] as $p => $path_location) {
+          $file_path_temp =isset($path_location)?(str_replace($package . '/' . $package . '.info', '', $path_location)):'';
+          if (($file_path_temp =='') && ($versions['package_type'] == 'module')) {
+            $file_paths[] = $drupal_docroot_path . "/sites/all/modules";
+          }
+          else {
+            $file_paths[] = ($file_path_temp !='')?realpath($file_path_temp):$drupal_docroot_path;
+          }
+        }
+        $git_commit_message['file_path'] =$file_paths;
+      }
+      else {
+        $file_path =isset($package_info_files[$package . '.info'])?(str_replace($package . '/' . $package . '.info', '', $package_info_files[$package . '.info'])):'';
```

Comment:
Should be `= isset`. Also, you need a whitespace before and after your ternary operator.